js/bundle/src/decoder.ts

```diff
@@ -0,0 +1,169 @@
+import * as CBOR from 'cbor';
+
+export class Bundle {
```

Comment:
Can we have JSDoc comments for this?

Reply:
Added, although I'm not really sure if we should require JSDocs in this package.